### PR TITLE
ansible: create a libvirt slave playbook

### PR DESCRIPTION
**Pull request by alfredodeza**
Signed-off-by: Alfredo Deza <adeza@redhat.com>

### CODE REVIEW

File: ansible/examples/slave_libvirt.yml [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+  tasks:
+
+    # vagrant doesn't have repositories, this chacra repo will be better to have
+    # around and can get updates as soon as a new vagrant version is published via
+    # chacractl
+    - name: add the vagrant repository
+      sudo: yes
+      apt_repository:
+        repo: "deb [trusted=yes] https://chacra.ceph.com/r/vagrant/latest/HEAD/ubuntu/xenial/flavors/default/ main"
```
Comment (andrewschoen) [inline ansible/examples/slave_libvirt.yml]: It's a bummer ``apt_repository`` doesn't take a url so you could use: https://chacra.ceph.com/repos/vagrant/latest/HEAD/ubuntu/xenial/repo/

File: ansible/examples/slave_libvirt.yml [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        # jenkins-job-builder job:
+        - libyaml-dev
+        - libvirt
+        - qemu-kvm
+        - libvirt-bin
+        - vagrant
+      when: ansible_pkg_mgr  == "apt"
+
+    - name: install the vagrant-libvirt plugin
+      shell: vagrant plugin install vagrant-libvirt
```
Comment (andrewschoen) [inline ansible/examples/slave_libvirt.yml]: will this install ``vagrant-libvirt`` for the ``jenkins-build`` user or ``ubuntu``?  Does that even matter?

Comment (alfredodeza) [inline ansible/examples/slave_libvirt.yml]: oh good point, I think it does install it locally per user

Comment (andrewschoen) [inline ansible/examples/slave_libvirt.yml]: I think you need to move this task down after the task that creates the ``jenkins_user``.

**Pull request merged**